config/initializers/dartsass.rb

```diff
@@ -0,0 +1,11 @@
+# config/initializers/dartsass.rb
+APP_STYLESHEETS = {
```

Comment:
Another small comment. I think you can remove the comment on line 1 (apologies, I know that it might be included in the docs in which case I'll also remove it from there)